README.md

```diff
@@ -1,8 +1,80 @@
-# OSV-Scanner Github Action
+# OSV-Scanner GitHub Action
 
-OSV-Scanner is offered as a GitHub Action. We currently have two different GitHub Actions:
+[OSV-Scanner](www.github.com/google/osv-scanner) offers two GitHub Actions:
 
 1. An action that triggers a scan with each pull request and will only check for new vulnerabilities introduced through the pull request.
 2. An action that performs a single vulnerability scan, which can be configured to scan on a regular schedule, or used as a check on releases to prevent releasing with known vulnerabilities in dependencies.
 
-## TODO
+Both actions are free for all repositories. 
+
+________
+[OSV-Scanner](#osv-scanner)
+
+[Installation](#installation)
+- [Automatic installation](#automatic-installation)
+- [Manual installation](#manual-installation)
+
+
+
+
+________
+
+## OSV-Scanner
+[OSV-Scanner](https://google.github.io/osv-scanner) provides an officially supported frontend to the [OSV database](https://osv.dev) that connects a project’s list of dependencies with the vulnerabilities that affect them. Since the OSV database is open source and distributed, it has several benefits in comparison with closed source advisory databases and scanners:
+
+- Each advisory comes from an open and authoritative source (e.g. the [RustSec Advisory Database](https://github.com/rustsec/advisory-db))
+- Anyone can suggest improvements to advisories, resulting in a very high quality database
+- The OSV format unambiguously stores information about affected versions in a machine-readable format that precisely maps onto a developer’s list of packages
+
+
+## Installation
+
+OSV-Scanner has a primary GitHub action that scans the repository and can be configured to scan on a schedule or on release. OSV-Scanner also has a secondary GitHub Action that performs a scan on pull request and reports newly introduced vulnerabilities. 
+
+The primary GitHub Action can be installed [automatically](#automatic-installation) or [manually](#manual-installation). 
+
+The secondary GitHub Action that runs on pull request must be installed [manually](#manual-installation)
+
+### Automatic installation
+
+1) From your GitHub project's main page, click “Actions” tab in the navigation bar.
```

Comment:
The steps here deviate from what is shown on Scorecard Action, but those steps are stale now. This has fewer clicks and is fresher.